### PR TITLE
Upgrade doc

### PR DESCRIPTION
**Pull request by babsey**
Upgrade doc for v3.2

### CODE REVIEW

**Review by jens-br — CHANGES_REQUESTED** [review]
Those are nice additions! I would suggest to inlcude some changes.

As a general aspect, we could think of some structural changes to the usage guides and the reference section - in the future.

File: docs/about/events.rst [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+======
+
+2022
+----
+
+:16 Jul: NEST Desktop is part of the onsite tutorial "T1: From single-cell modeling to large-scale network dynamics with NEST Simulator." of `CNS 2022 <https://www.cnsorg.org/cns-2022-tutorials>`__, presented by Jasper Albers, Pooja Babu and Charl Linssen, Melbourne, Australia.
+:31 Jun: Satellite tutorial at `CNS <https://hackmd.io/W09z_RysTUWuwpHtgphdxQ?view>`__, online.
+:23 - 24 Jun: Workshop "NEST Desktop: A “Let’s Play Together” for neuroscience" and Poster "NEST Desktop: Explore new frontiers" at `NEST Conference <https://events.hifis.net/event/305/>`__, online.
+:13 - 15 Jun: Workshop at `EBRAINS BASSES <https://www.humanbrainproject.eu/en/education/ebrains-workshops/basses/>`__, presented by Johanna Senk, Rome, Italy.
+:26 - 28 Apr: MSc course "Biophysics of Neurons and Networks" at BCF [#f1]_, Freiburg, Germany.
```
[the remark this answers is not in the document]
Comment (babsey) [reply docs/about/events.rst:11]: Indeed, the footnote link is not good styled.
The style of sphinx material conflicts with bootstrap css.

File: docs/about/index.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+.. toctree::
+   :maxdepth: 1
+
+   events
+   releases
+   nuspic
+
+
+.. include:: abstract.rst
```
[the remark this answers is not in the document]
Comment (babsey) [reply docs/about/index.rst:13]: Be honest, I do not like the iframe of the live-demo. 
Instead, I prefer using link to open a live-demo in a new tab.

File: docs/developer/coding-conventions.rst [hunk 3]
```diff
@@ -23,7 +23,7 @@ General coding conventions
   (e.g when they contain very long URLs or within bullet points, \
   the latter with the possibility to use backslashes, which is sometimes quite cumbersome).
 - The general coding conventions for Vue, TypeScript \
-  and for Python (see `PEP 8 <https://www.python.org/dev/peps/pep-0008/>`__) should be followed.
+  and for Python (|see| `PEP 8 <https://www.python.org/dev/peps/pep-0008/>`__) should be followed.
```
[the remark this answers is not in the document]
Comment (babsey) [reply docs/developer/coding-conventions.rst:26]: Fixed.

File: docs/lecturer/master/network-dynamics.rst [hunk 4]
```diff
@@ -51,7 +51,7 @@ A typical value for the connection probability in local cortical networks is 10%
 
    .. note::
       There are several different options to wire up a network randomly
-      (see https://www.nest-simulator.org/connection-management).
+      (|see| https://www.nest-simulator.org/connection-management).
```
[the remark this answers is not in the document]
Comment (babsey) [reply docs/lecturer/master/network-dynamics.rst:54]: Fixed

File: docs/user/setup/snap.rst [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+.. image:: /_static/img/logo/snapcraft-logo.png
+   :align: left
+   :target: #snap-linux
+   :width: 120px
+
+You can download NEST Desktop via snap.
+
+.. code-block:: bash
+
+   snap install nest-desktop
```
[the remark this answers is not in the document]
Comment (babsey) [reply docs/user/setup/snap.rst:13]: Yeah, I am aware of it. Tried several times. We can create an issue of it, with low priority.

File: docs/troubleshootings/error.rst [hunk 6]
```diff
@@ -0,0 +1,38 @@
+.. _error-server-not-found:
+
+Server not found
+----------------
+
+NEST Desktop cannot find the NEST Simulator.
+It has two possible reasons:
+
+- NEST Desktop has a wrong URL under which it tries to contact the server.
+  |see| the :doc:`FAQ for NEST Simulator </troubleshootings/nest-simulator>`.
```
[the remark this answers is not in the document]
Comment (babsey) [reply docs/troubleshootings/error.rst:10]: fixed.

File: docs/lecturer/bachelor/single-neuron-poisson-input.rst [hunk 7]
```diff
@@ -34,7 +34,7 @@ You can now employ the Poisson generator functionality of NEST to explore the re
    the other one consisting of inhibitory neurons.
```
[the remark this answers is not in the document]
Comment (babsey) [reply docs/lecturer/bachelor/single-neuron-poisson-input.rst:35]: For now, we can ignore these warnings.

File: docs/user/setup/appimage.rst [hunk 8]
```diff
@@ -0,0 +1,12 @@
+AppImage |linux|
```
[the remark this answers is not in the document]
Comment (jens-br) [reply docs/user/setup/appimage.rst:1]: Let's address it in an issue (#524 ).

File: docs/user/setup/apptainer.rst [hunk 9]
```diff
@@ -0,0 +1,67 @@
+Apptainer |linux|
```
[the remark this answers is not in the document]
Comment (jens-br) [reply docs/user/setup/apptainer.rst:1]: (#524 )

File: docs/user/setup/docker-compose.rst [hunk 10]
```diff
@@ -0,0 +1,60 @@
+Docker Compose |linux| |windows| |apple|
```
[the remark this answers is not in the document]
Comment (jens-br) [reply docs/user/setup/docker-compose.rst:1]: (#524 )

File: docs/user/setup/conda.rst [hunk 11]
```diff
@@ -0,0 +1,72 @@
+Conda |linux| |windows| |apple|
```
[the remark this answers is not in the document]
Comment (jens-br) [reply docs/user/setup/conda.rst:1]: (#524)

File: docs/user/setup/docker-compose.rst [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+   :target: #docker-compose-linux-windows-apple
+   :width: 120px
+
+Docker is a virtualization software packaging applications and its dependencies.
+Docker Compose is a tool for running multi-container applications on Docker defined using the Compose file format.
+To get more information, see the `official page of Docker Compose <https://github.com/docker/compose>`__.
+
+|br|
+
+Windows |windows| and macOS |apple|
```
[the remark this answers is not in the document]
Comment (jens-br) [reply docs/user/setup/docker-compose.rst:15]: (#524)

File: docs/lecturer/bachelor/single-neuron-poisson-input.rst (hunk 7 above)
[the remark this answers is not in the document]
Comment (jens-br) [reply docs/lecturer/bachelor/single-neuron-poisson-input.rst:35]: We will track it in #524 .

**Review by jens-br — APPROVED** [review]
LGTM :+1:

**Pull request merged**